src/App.js

```diff
@@ -9,7 +10,13 @@ const article = {
 };
 
 const App = () => (
-  <h1>React Article</h1>
+  < >
+    <Article
+      title={article.title}
```

Comment:
You can use destructuring for article object before App component. After that you can leave only title = {article}